doc/minimum-weight-spanning-tree.adoc

```diff
@@ -75,17 +152,50 @@ image::{img}/mst_result.png[]
 .Running algorithm and writing back results
 [source,cypher]
 ----
-CALL algo.spanningTree(label:String, relationshipType:String, weightProperty:String, startNodeId:long, {writeProperty:String}) YIELD loadMillis, computeMillis, writeMillis, weightSum, effectiveNodeCount
+CALL algo.spanningTree(label:String, relationshipType:String, weightProperty:String, startNodeId:int, {writeProperty:String}) 
```

Comment:
Is this an accidental change? In Java land this number is a `long` so I think it makes sense to use those types in the docs.

Reply:
We were using only int and float in docs till now so we do not differentiate between int and long as for end user it shouldn't make any difference